Example/Tests/Core/Network/Api/AwesomeAdsApiTests.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  AwesomeAdsApiTests.swift
+//  Tests
+//
+//  Created by Gunhan Sancar on 16/04/2020.
+//  Copyright © 2020 Gabriel Coman. All rights reserved.
+//
+
+import XCTest
+import Nimble
+import Moya
+import Mockingjay
+@testable import SuperAwesome
+
+class AwesomeAdsApiTests: XCTestCase {
+    private var provider: MoyaProvider<AwesomeAdsTarget>!
+    private var ad: Ad? = nil
+    private var error: Error?  = nil
+    private let mockQuery = AdQuery(test: true, sdkVersion: "", rnd: 1, bundle: "",
+                        name: "", dauid: 1, ct: .wifi, lang: "", device: "",
+                        pos: 1, skip: 1, playbackmethod: 1, startdelay: 1, instl: 1, w: 1, h: 1)
+    
+    override func setUp() {
+        super.setUp()
+        provider = MoyaProvider<AwesomeAdsTarget>(plugins:[NetworkLoggerPlugin()])
+        ad = nil
+        error = nil
+    }
+    
+    private func prepareResponse(json: String) {
+        // Given
+        let placementId = 1
```

Comment:
I don't know about you, I like to be quite strict with my number declarations in Swift as this can mean a double or an int. I realise it is meant to be an int here, so i would help the compiler out where ever possible by typing: 
```suggestion
        let placementId: Int = 1
```

Or if it was meant to be a double:
```suggestion
        let placementId = 1.0
```